### DISCUSSION TITLE
Quarto hangs after knitting

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#8479] [Q&A]
### Description

I have a website project with many qmd files, mostly with R code. It used to render fine with Quarto 1.3.X. Now after updating to 1.4.549, Running `quarto render` on the whole project seems to hang on some qmds like this:

```
...
19/25                   
20/25 [unnamed-chunk-10]
21/25                   
22/25 [unnamed-chunk-11]
23/25                   
24/25 [unnamed-chunk-12]
25/25                   
output file: index.knit.md
```

But, if I try `quarto render path/bla.qmd` specifically on that file, it renders and produces html just fine.

Any ideas on how to troubleshoot this? What could be wrong? I am unsure how to create a reproducible example.

`quarto 1.4.549`

**Comment by cderv** [comment]
For more troubleshooting, you can look at https://quarto.org/docs/troubleshooting. Especially, verbose mode and stack trace.

If you can share you project (open github repo maybe ?), then I can try look at it. A lot happens in Quarto, even after **knitr** step, so it can come from other things. 

We need an example to look into that. Thanks !

**Comment by royfrancis** [comment]
The issue was on a Mac M2. I tried the same code and same version on an Ubuntu set up and it worked. So I went back to the Mac, reinstalled Quarto and docker and then it worked. So, must have been some weird system issue. It coincided with an OS update.
